apps/dashboard/src/components/workflows-data-table.tsx

```
@@ -0,0 +1,437 @@
+} from "@/components/ui/dropdown-menu";
+import { Label } from "@/components/ui/label";
+import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "@/components/ui/select";
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from "@/components/ui/table";
+import { useWorkflowExecutions } from "@/hooks/useWorkflowExecutions";
+
+interface WorkflowData {
+    id: string;
+    name: string;
+    description?: string | null;
+    configuration: {
+        fields?: any[];
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)